wagtail_localize/management/commands/import_po.py

```diff
@@ -0,0 +1,135 @@
+    """Recursively browse, export a page and yield its sub pages."""
+    yield page
+    for child in page.get_children():
+        for sub in get_pages(child):
+            yield sub
+
+
+class CommandHandler:
+    def __init__(self, site, potfile, po_outfmt):
+        page = site.root_page
+
+
+class Command(BaseCommand):
+    help = "Export .po files for all locals"
+
+    def add_arguments(self, parser):
+        po_outfmt = (
+            './locales/{language_code}/LC_MESSAGES/'
+            + settings.WAGTAIL_SITE_NAME
+            + '.po'
+        )
+        parser.add_argument(
+            '--site',
+            dest='site',
+            type=str,
+            help="host:port of the site, the default site will be exported "
+            "if not specified",
+        )
+        parser.add_argument(
+            '--po-fmt', dest='po_outfmt', default=po_outfmt, type=str
+        )
+        parser.add_argument(
+            '--pot-locale',
+            dest='pot_locale',
+            type=str,
+            help="Override the defaut locale to force the pivot language used"
+            "to generate the .pot file",
+        )
+
+        parser.add_argument(
+            '--publish',
+            dest='publish',
+            action='store_const',
+            const=True,
+            default=False,
+        )
+
+    def handle(self, **options):
+
+        if options.get('site'):
+            host_port = option.get('split').rsplit(':', maxsplit=1)
+            site = Site.objects.filter(
+                hostname=host_port[0],
+                port=int(host_port[1]) if len(host_port) == 2 else 80,
+            ).first()
+        else:
+            site = Site.objects.filter(is_default_site=True).first()
+
+        pot_locale = options.get('pot_locale')
+        if pot_locale:
+            source_locale = Locale.objects.get(language_code=pot_locale)
+        else:
+            source_locale = Locale.objects.get(id=default_locale_id())
+
+        locales = Locale.objects.filter(is_active=True).exclude(
+            id=source_locale.id
+        )
+
+        publish = options.get('publish', False)
+
+        root_page = site.root_page if site else None
+        if root_page is None:
+            raise ValueError("Site is not properly configured, no root page.")
+
+        for target_locale in locales:
+            print(
+                "Loading translations for {} -> {}".format(
+                    source_locale, target_locale
+                )
+            )
+            translations = LocaleLoader(target_locale, options)
+            message_ingestor = MessageIngestor(
+                source_locale, target_locale, translations.items,
+            )
+
+            for page in get_pages(root_page):
+                print("Loading page {}".format(page))
+                instance = page.get_latest_revision().as_page_object()
+                if instance.is_source_translation:
+                    if instance.locale.id != source_locale.id:
```

Comment:
I have an issue with that, in fact the pages are localized, and I don't find how it works, is it the `LANGUAGE_CODE` from the settings when it the webpage was created ?

I don't find any way to customize it. I want to keep the `en` as the django lang for my command line and for the admin,
but not for the wesite published (`sub-app` if it is called like this).